src/components/alert/styled.ts

```diff
@@ -0,0 +1,75 @@
+import stitches from '../../stitches.config';
+
+const { styled } = stitches;
+
+const setColors = (bgColor: string, color: string) => ({
+  backgroundColor: bgColor,
+  borderColor: color,
+
+  svg: {
+    fill: color,
+  },
+});
+
+export const AlertStyled = styled('div', {
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  padding: '8px 16px 8px 12px',
```

Comment:
```suggestion
  padding: '8px 16px 8px 12px',
```

use spacing function